### DISCUSSION TITLE
希望增加按住辅助键如Alt时临时禁用右键手势，50感谢费

### DISCUSSION THREAD

**Discussion by moonhuahua** [shuax/MouseInc.Settings#99] [Ideas]
希望增加按住辅助键如Alt时临时禁用右键手势，肯定感谢，松开自动启用鼠标手势

**Comment by moonhuahua** [comment]
> 希望增加按住辅助键如Alt时临时禁用右键手势，肯定感谢，松开自动启用鼠标手势

PS软件或其它软件里面会经常用的到的

**Reply by JzCherry** [reply]
2023年了，看来是没希望了

**Comment by shuax** [comment]
当前软件不适合鼠标手势建议画Z添加到排除列表。

**Reply by iverjun** [reply]
不不不,PS很适合用手势,特别是在手绘版的情况下.我也十分希望加入ALT临时禁用右键手势

**Reply by iverjun** [reply]
本来我是用Strokelt,但它已经不更新了,
![Snipaste_2021-09-29_09-46-12](https://user-images.githubusercontent.com/91578245/135189115-8ebb6b89-2b86-43c2-9b4a-922a92b239e9.jpg)

**Reply by iverjun** [reply]
![Snipaste_2021-09-29_10-11-48](https://user-images.githubusercontent.com/91578245/135191440-3063c5bd-8606-4c0a-9e4e-65fa41079a1e.jpg)
